apps/web/src/app/[locale]/tracks/_components/enrolled-tracks-skeleton.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from '@repo/ui/components/card';
+import { Skeleton } from '@repo/ui/components/skeleton';
+import En from '~/locales/en';
```

Comment:
`En` not used. @bautistaaa maybe we can add eslint rule for unused imports?

Reply:
ill make a GH issue

Reply:
ill make a GH issue